### DISCUSSION TITLE
Xiangqi tournaments with Pikafish. Where to find similar level rivals?

### DISCUSSION THREAD

**Discussion by nguyenpham** [official-pikafish/Pikafish#24] [Show and tell]
AMD Ryzen 7 1700 Eight-Core Processor
64 MB Hash
Timer: 40/300+1
Total 300 games Roundrobin
GUI: MoonRiver

![r](https://user-images.githubusercontent.com/1370755/209333175-095daf68-534a-40a5-aae7-577295870ff1.png)

Pikafish has beaten all other rivals by large margins.
- Any idea about the ELO of Pikafish or anyone in the tournament? I will use it to calculate the ELO of other engines
- I start looking for new close-level rivals for Pikafish. Any suggestion?

**Comment by Quocvuong82** [comment]
Chú Nguyên ơi chú có thể chia sẻ GUI MoonRivers không ạ?
Cháu cảm ơn chú.

**Reply by nguyenpham** [reply]
MoonRiver will be released as freeware so soon!

**Reply by Quocvuong82** [reply]
Chú Nguyên ơi, qua việc trải nghiệm GUI MoonRiver cháu thấy rất tuyệt vời, tuy nhiện có một số vấn đề thế này ạ:
1) Chi chọn hình quân cờ từ những style SVG thì không load được hình quân Mã và quân Tượng
https://thptchuyensonlaeduvn-my.sharepoint.com/:i:/g/personal/vuong03_thptchuyensonla_edu_vn/ERuY9ZCNMyhGmfnIAlU5d4UBG5cdChz07QUTVtp2H5XbCg?e=2rxJ0y

2) Khi tổ chức Tour, việc GUI kết luận kết quả ván cờ có nhiều vấn đề lắm ạ

Cháu mong chú có thể đọc được tin nhắn này và xem lại một số vấn đề đó, mong phiên bản mới của MoonRiver ạ.

**Reply by mieucun123** [reply]
> Chú Nguyên ơi, qua việc trải nghiệm GUI MoonRiver cháu thấy rất tuyệt vời, tuy nhiện có một số vấn đề thế này ạ:
> 
> 1. Chi chọn hình quân cờ từ những style SVG thì không load được hình quân Mã và quân Tượng
>    https://thptchuyensonlaeduvn-my.sharepoint.com/:i:/g/personal/vuong03_thptchuyensonla_edu_vn/ERuY9ZCNMyhGmfnIAlU5d4UBG5cdChz07QUTVtp2H5XbCg?e=2rxJ0y
> 2. Khi tổ chức Tour, việc GUI kết luận kết quả ván cờ có nhiều vấn đề lắm ạ
> 
> Cháu mong chú có thể đọc được tin nhắn này và xem lại một số vấn đề đó, mong phiên bản mới của MoonRiver ạ.

Bác cho em link tải moonriver được không ?
Em thấy bác nguyenpham không cập nhật thông tin gì về moonriver

**Comment by PikaCat-OuO** [comment]
ELO of pikafish is not quite sure.
I think close level rivals will be commercial engines.

**Comment by nguyenpham** [comment]
Found a first same-level rival, named CCStockfish. An open-source. Not sure about the relationship with Pikafish since it listed the Pikafish team in the author's info. Is that a clone but using a different net?

On the attached picture, it has a close point to Pikafish. The result of direct matches (vs Pikafish, 200 games) are totally equal: 2 wins, 2 losses, the rest are drawn.

![mr05](https://user-images.githubusercontent.com/1370755/209597356-72346752-40f7-4df8-bde3-d42565d7e765.PNG)

**Reply by PikaCat-OuO** [reply]
CCStockfish is a copy paste of Pikafish really when you compare the code XD. It uses a different net and is weaker.
CCStockfish is clone by that guy who made ggzero, our community is not in good relationship with him.

**Reply by agogugu** [reply]
> Found a first same-level rival, named CCStockfish. An open-source. Not sure about the relationship with Pikafish since it listed the Pikafish team in the author's info. Is that a clone but using a different net?
> 
> On the attached picture, it has a close point to Pikafish. The result of direct matches (vs Pikafish, 200 games) are totally equal: 2 wins, 2 losses, the rest are drawn.
> 
> ![mr05](https://user-images.githubusercontent.com/1370755/209597356-72346752-40f7-4df8-bde3-d42565d7e765.PNG)

Why aren't u using the latest version of Pikafish?

**Comment by nguyenpham** [comment]
Another tournament. Almost the same setup as the previous one. Just a bit longer opening lines (14 plies instead of 10).

Of 200 direct matches (Pikafish vs CCStockfish), Pikafish won 3, and the rest (297 games) are drawn.

![mr06](https://user-images.githubusercontent.com/1370755/209722207-57ce8aa4-28a8-482f-9385-a05fd1b3f66d.PNG)

**Comment by clemtkx** [comment]
Hello, how do i run tournaments with engines such as above and have the ability to view the moves later on ?
